### PR TITLE
Create core-notes-2020-09-25.md

### PR DESCRIPTION
**Pull request by ppannuto**
Action item: @alevy will organize USB/CTAP call

----

### Pull Request Overview

Notes from the call

### Testing Strategy

N/A

### TODO or Help Wanted

N/A

### Documentation Updated

- [x] Updated the relevant files in `/docs`, or no updates are required.

### Formatting

- [ ] Ran `make prepush`.

### CODE REVIEW

**Review by hudson-ayers — DISMISSED** [review]

**Review by alistair23 — APPROVED** [review]

File: doc/wg/core/notes/core-notes-2020-09-25.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+ * Phil: Timer merged!!
+ * Brad: elf2tab rounds application outputs to power-of-2, because of Cortex-M MPU limitations. For Risc-V, this power-of-2 isn't a requirement, so looking at moving the logic out of elf2tab
+     * Amit: Sounds reasonable, perhaps tockloader responsibility?
+     * Brad: Actually, it is already a tockloader responsibility. The multiple compiled binaries already a major R5 complexity
+     * Amit: Yeah, seems like a pre-tockloader choice whose time has come to move
+ * Johnathan: OpenSK is working on adding NFC support for 52840; also working on CryptoCell, but that's moving slower b/c very complex
+
+## Interrupt control semantics in Tock/HIL
+ * A: Summary: Who in the kernel is responsible to enable/disable NVIC on ARM / equiv on R5?
+ * P: Not just about who, but about what happens
```
Comment (jrvanwhy) [inline doc/wg/core/notes/core-notes-2020-09-25.md:26]: I think we need to be a bit more clear that "A" is Amit and "P" is Phil.

**Review by jrvanwhy — CHANGES_REQUESTED** [review]

File: doc/wg/core/notes/core-notes-2020-09-25.md (hunk 1 above)
Comment (brghena) [reply doc/wg/core/notes/core-notes-2020-09-25.md:26]: Yeah, maybe do a find/replace on all of these name abbreviations

Comment (ppannuto) [reply doc/wg/core/notes/core-notes-2020-09-25.md]: Expanded out all the names ✔️

**Review by lschuermann — APPROVED** [review]

**Review by jrvanwhy — APPROVED** [review]

**Review by hudson-ayers — APPROVED** [review]
bors r+

**Pull request merged**